static/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -348,17 +350,25 @@ export default class AbstractExternalIssueForm<
                     ...fields,
                     noOptionsMessage: () => 'No options. Type to search.',
                   }))
-                  .map(field => (
-                    <FieldFromConfig
-                      disabled={this.state.reloading}
-                      field={field}
-                      flexibleControlStateSize
-                      inline={false}
-                      key={`${field.name}-${field.default}-${field.required}`}
-                      stacked
-                      {...this.getFieldProps(field)}
-                    />
-                  ))}
+                  .map(field => {
+                    return (
+                      <React.Fragment
+                        key={`${field.name}-${field.default}-${field.required}`}
```

Comment:
This is probably better off being index based right? `field.default` and `field.required` don't really have string-friendly values to use as a React Component key.

Reply:
You could use a hybrid of `field.name` and `index`. Or we could have a broader discussion on inforcing unique `name` fields on the schema so that you could just leave it as `field.name`.

Reply:
Looking at this again it looks like you just rearranged props, and didn't introduce this change, though I still stand by my point!